common/util/autoware_debug_tools/src/pose2tf.py

```diff
@@ -0,0 +1,68 @@
+#! /usr/bin/env python3
+
+# Copyright 2020 Tier IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import argparse
+import sys
+import rclpy
+from rclpy.node import Node
+import tf2_ros
+from geometry_msgs.msg import PoseStamped
+
+
+class Pose2TfNode(Node):
+    def __init__(self, options):
+        super().__init__('pose2tf')
+        self._options = options
+        self._tf_broadcaster = tf2_ros.TransformBroadcaster(self)
+        self._sub_pose = self.create_subscription(
+            PoseStamped, self._options.topic_name, self._on_pose, 1)
+
+    def _on_pose(self, msg):
+        try:
+            (trans, quat) = Pose2TfNode.create_transform(msg.pose)
+            self._tf_broadcaster.sendTransform(
```

Comment:
sendTransform function has only 1 argument.
Plaese check the following code.
/opt/ros/foxy/lib/python3.8/site-packages/tf2_ros/transform_broadcaster.py

This is C++ code, but it's also helpful
https://github.com/tier4/AutowareArchitectureProposal.iv/blob/master/localization/pose_twist_fusion_filter/ekf_localizer/src/ekf_localizer.cpp#L200-L213

Reply:
fixed!